### ISSUE TITLE
development build fails for K8

### ISSUE THREAD

**Issue by sehz** [infinyon/fluvio#1207]
```make TEST_ARG_DEVELOP=--develop smoke-test-k8```

This result in:
```
thread 'main' panicked at 'Failed to install k8 cluster: InstallK8(UnableToDetectService)', tests/runner/src/utils/setup/environment/k8.rs:56:14
```

**Comment by nicholastmosher** [comment]
This happens because of not building the minikube image, I think it should be solved by https://github.com/infinyon/fluvio/pull/1204

I think we should also set `TEST_ARG_DEVELOP=--develop` as default, can we do that in #1204 also?

**Comment by sehz** [comment]
This is fixed with #1204. I will add `--develop` to `TEST_ARG_DEVELOP`

**Comment by sehz** [comment]
works now.
